### DISCUSSION TITLE
How to interpolate directly to 60fps?

### DISCUSSION THREAD

**Discussion by IrodoriCiel** [mafiosnik777/enhancr#39] [Q&A] [answered]
How to interpolate into 60 fps directly and not only 2x?

I would like to interpolate a video in 24 fps to 60 fps?, but the output is only to 48 fps, i could to interpolate the output video again but it will have 96 fps

**Comment by ThatPlasma** [comment, marked as answer]
with my knowledge of RIFE, it can be done the question is if it gets implemented into enhancer 
(side note RIFE was created around 2/3/4x and may cause issues with something like 60fps (what those issues are idk as I haven't tried it my self)
